app/build.gradle

```diff
@@ -127,47 +127,48 @@ androidExtensions {
 
 ext {
     // App dependency versions
-    supportVersion = '27.1.1'
-    playServicesVersion = '15.0.1'
-    daggerVersion = '2.14.1'
-    retrofitVersion = '2.3.0'
-    okHttpVersion = '3.9.1'
+    supportVersion = '1.0.0-beta01'
+    playServicesVersion = '16.1.0'
+    daggerVersion = '2.20'
+    retrofitVersion = '2.4.0'
+    okHttpVersion = '3.11.0'
     leakcanaryVersion = '1.5.4'
 
     // Test dependency versions
-    mockitoVersion = '2.13.0'
-    robolectricVersion = '3.7.1'
-    androidTestSupportVersion = '1.0.1'
-    espressoVersion = '3.0.2'
+    mockitoVersion = '2.25.0'
+    robolectricVersion = '4.2'
+    androidTestSupportVersion = '1.1.0'
+    espressoVersion = '3.1.0-alpha4'
 }
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    kapt "com.android.databinding:compiler:$rootProject.ext.android_plugin_version"
+    kapt "androidx.databinding:databinding-compiler:$rootProject.ext.android_plugin_version"
     // Need this because of Kotlin
 
     // Android/Google libraries
-    implementation 'androidx.core:core-ktx:0.3'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
-    implementation "com.android.support:appcompat-v7:$supportVersion"
-    implementation "com.android.support:recyclerview-v7:$supportVersion"
-    implementation "com.android.support:cardview-v7:$supportVersion"
-    implementation "com.android.support:support-annotations:$supportVersion"
-    implementation "com.android.support:design:$supportVersion"
+    implementation 'androidx.core:core-ktx:1.0.1'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation "androidx.appcompat:appcompat:1.1.0-alpha02"
+    implementation "androidx.recyclerview:recyclerview:1.1.0-alpha02"
```

Comment:
If there is an option between the alpha dependency and a stable one, let's try to prefer the stable one for the dependencies.